Plugins/Locales/deDE.lua

```diff
@@ -2,9 +2,9 @@ local L = BigWigsAPI:NewLocale("BigWigs: Plugins", "deDE")
 if not L then return end
 
 L.comma = ", "
---L.width = "Width"
---L.height = "Height"
---L.sizeDesc = "Move the slider if you need to set an exact size. You can also type the value into the box, which has no maximum."
+L.width = "Breite"
+L.height = "Höhe"
+L.sizeDesc = "Bewege den Schieber, falls du eine exakte Größe festlegen möchtest. Du kannst ebenfalls eine Zahl in das Feld eintragen, diese Zahl kann beliebig groß sein."
```

Comment:
`L.sizeDesc`
Unfortunately the English commented out version for this was wrong in the translation file (out of date). Could you re-translate this using the translation that's in the enUS file?